.github/workflows/build_project.yml

```
@@ -15,7 +15,7 @@ jobs:
             - uses: actions/checkout@v3
             - uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)